rosdep/python.yaml

```diff
@@ -3709,6 +3709,7 @@ python-requests:
       packages: [requests]
   ubuntu:
     '*': [python-requests]
+    bionic_python3: [python3-requests]
```

Comment:
We've deprecated this method of doing python3 keys; we found it did not work very well.  Instead, we are now using separate keys for python2 and python3; see https://github.com/ros/rosdistro/blob/4a422638d8506f2930bbd2d00993a0522425fe72/rosdep/python.yaml#L5325 for one example on how to do it.  The `python3-requests` is indeed missing, so please make it look like the other python3 keys.  Thanks.

Reply:
Thanks, see the new changes. I moved the other `python3-requests` entries to conform to this new way as well. Let me know if that is ok or if you want me to change those back.